openapi/openapi/src/main/scala/endpoints4s/openapi/Endpoints.scala

```diff
@@ -29,8 +29,9 @@ trait EndpointsWithCustomErrors
     for (e <- endpoints) {
       val key = e.path
       val newValue = pathItems.get(key) match {
-        case Some(current) => PathItem(current.operations ++ e.item.operations)
-        case None          => PathItem(e.item.operations)
+        case Some(current) =>
+          PathItem(collection.immutable.ListMap(current.operations.toSeq: _*) ++ e.item.operations)
+        case None => PathItem(collection.immutable.ListMap(e.item.operations.toSeq: _*))
```

Comment:
What do you think of moving this to the implementation of `PathItem`? Ie, you could change the type of `operations` to be a `SeqMap`.

Reply:
didn't want to break compatibility

Reply:
I believe this can be done without breaking the compatibility, no? Since `operations` is private in `PathItem`.

Reply:
withOperations is public

Reply:
Good point. And actually `operations` is public. Only the constructor is private…